skimage/segmentation/tests/test_random_walker.py

```diff
@@ -60,6 +61,9 @@ def make_3d_syntheticdata(lx, ly=None, lz=None):
     return data, seeds
 
 
+@pytest.mark.filterwarnings(
+    "ignore:upcasting `data` with dtype float16 to float32::skimage"
```

Comment:
```suggestion
    "ignore:upcasting `data` with dtype float16 to float32:skimage"
```
Right?

Reply:
Nope. The [syntax](https://docs.python.org/3/library/warnings.html#describing-warning-filters) is `action:message:category:module:line`.